diff --git a/Makefile b/Makefile
index 045a62e..b31d10b 100644
--- a/Makefile
+++ b/Makefile
@@ -144,9 +144,9 @@ check: ${LIBIFUPDOWN_LIB} ${CMDS}
 	kyua test || (kyua report --verbose && exit 1)
 
 install: all
-	install -D -m755 ${MULTICALL} ${DESTDIR}${SBINDIR}/${MULTICALL}
+	install -D -m755 ${MULTICALL} ${DESTDIR}${PREFIX}${SBINDIR}/${MULTICALL}
 	for i in ${CMDS}; do \
-		ln -s ${SBINDIR}/${MULTICALL} ${DESTDIR}${SBINDIR}/$$i; \
+		ln -s ${DESTDIR}${PREFIX}${SBINDIR}/${MULTICALL} ${DESTDIR}${PREFIX}${SBINDIR}/$$i; \
 	done
 	for i in ${EXECUTOR_SCRIPTS}; do \
 		install -D -m755 executor-scripts/${LAYOUT}/$$i ${DESTDIR}${EXECUTOR_PATH}/$$i; \
