diff --git a/Makefile b/Makefile
index 23e072f..93634e2 100644
--- a/Makefile
+++ b/Makefile
@@ -248,7 +248,7 @@ install: hikari hikari-unlocker share/man/man1/hikari.1
 	mkdir -p ${DESTDIR}/${PREFIX}/share/backgrounds/hikari
 	mkdir -p ${DESTDIR}/${PREFIX}/share/wayland-sessions
 	mkdir -p ${DESTDIR}/${ETC_PREFIX}/etc/hikari
-	mkdir -p ${DESTDIR}/${ETC_PREFIX}/etc/pam.d
+	mkdir -p ${DESTDIR}/${PREFIX}/lib/pam.d
 	sed "s,PREFIX,${PREFIX}," etc/hikari/hikari.conf > ${DESTDIR}/${ETC_PREFIX}/etc/hikari/hikari.conf
 	chmod 644 ${DESTDIR}/${ETC_PREFIX}/etc/hikari/hikari.conf
 	install -m ${PERMS} hikari ${DESTDIR}/${PREFIX}/bin
@@ -256,7 +256,7 @@ install: hikari hikari-unlocker share/man/man1/hikari.1
 	install -m 644 share/man/man1/hikari.1 ${DESTDIR}/${PREFIX}/share/man/man1
 	install -m 644 share/backgrounds/hikari/hikari_wallpaper.png ${DESTDIR}/${PREFIX}/share/backgrounds/hikari/hikari_wallpaper.png
 	install -m 644 share/wayland-sessions/hikari.desktop ${DESTDIR}/${PREFIX}/share/wayland-sessions/hikari.desktop
-	install -m 644 etc/pam.d/hikari-unlocker.${OS} ${DESTDIR}/${ETC_PREFIX}/etc/pam.d/hikari-unlocker
+	install -m 644 etc/pam.d/hikari-unlocker.${OS} ${DESTDIR}/${PREFIX}/lib/pam.d/hikari-unlocker
 
 uninstall:
 	-rm ${DESTDIR}/${PREFIX}/bin/hikari
